test/admin_ui/test_stats_drilldown.rb

```diff
@@ -13,6 +13,7 @@ def setup
     LogItem.clean_indices!
   end
 
```

Comment:
Please skip instead of only outcommenting. See f.e.: https://github.com/FIWARE/api-umbrella/blob/master/test/proxy/test_mongodb_replica_set.rb#L8